packages/components/form/src/form-control-mixin.ts

```diff
@@ -1,7 +1,6 @@
 import type { PropertyValues, ReactiveElement } from 'lit';
 import type { Constructor } from '@sl-design-system/shared';
 import { property, state } from 'lit/decorators.js';
-import styles from './form-control-mixin.scss.js';
 import { UpdateValidityEvent } from './update-validity-event.js';
```

Comment:
It looks like there is a description missing for the documentation part of `UpdateValidityEvent`

Reply:
I managed to add a description for the event, but the event name remains blank. I think we need a new story to "fix" the cem tool to output event metadata properly.

Reply:
Maybe it's possible to add a name somehow in the comment? :) We have it already live on the website so in my opinion it would be better to have the value somehow added here?

Reply:
Maybe it's possible to add a name somehow in the comment? :) We have it already live on the website so in my opinion it would be better to have the value somehow added here?